pallets/rmrk-equip/src/functions.rs

```diff
@@ -462,8 +465,26 @@ where
 						Err(Error::<T>::NoEquippableOnFixedPart.into())
 					},
 					PartType::SlotPart(mut slot_part) => {
-						// Update equippable value
-						slot_part.equippable = equippables;
+						match operation {
+							EquippableOperation::Add(equippable) => {
+								if let EquippableList::Custom(mut equippables) =
+									slot_part.equippable
+								{
+									let _ = equippables.try_push(equippable);
```

Comment:
What happens if this returns `Error` bc the BoundVec exceeds the length?

Reply:
Yeah, this should be handled, fixed it now.